src/components/CmsZone.vue

```diff
@@ -224,6 +241,29 @@ export default class CmsZone extends Vue {
     return (index: number) => `cms-zone-content-${this.zoneId}-${index}`;
   }
 
+  get zoneType(): string {
+    if (!this.lastResponse || this.zoneStatus === 'error') {
+      return '';
+    }
+    return this.lastResponse.zone_type;
+  }
+
+  get zoneHeader(): string {
+    return this.lastResponse && this.lastResponse.zone_header
```

Comment:
nit: i think this would work as just `this.lastResponse?.zone_header ? ...`?

Reply:
Yeah, that makes sense, will change.